app/assets/javascripts/_common.js

```diff
@@ -1,65 +1,72 @@
 $(document).ready(function() {
-  if ($('#login') && $('#username')) {
-    $('#username').focus();
+  if ($("#login") && $("#username")) {
+    $("#username").focus();
   }
 
-  $('.link_select').change(function(e) {
+  $(".link_select").change(function(e) {
     window.location = this.value;
   });
 
-  $('form#ajax_form').submit(function(e){
+  $("form#ajax_form").submit(function(e){
     e.preventDefault(); //Prevent the normal submission action
     var form = $(this);
+    var hideElementAttr = form.attr("data-hide-target");
+    var hideElement = $(`[${hideElementAttr}]`);
```

Comment:
would this work instead?  not a big deal, I just try to reach for "vanilla" JS when I can
`document.querySelectorAll("[data-hid-target]");`

Reply:
That doesn't work, as the `data-*` are not valid selectors to `querySelectorAll` (such as `.class` or `#id`).